packages/create-sitecore-jss/src/common/processes/next.ts

```diff
@@ -37,7 +37,7 @@ export const nextSteps = async (appName: string, nextStepsArr: string[]) => {
   console.log(chalk.white('                 / // /\\ \\_\\ \\  '));
   console.log(chalk.white('                 \\___/___/___/'));
   console.log();
-  console.log(`JSS application ${chalk.green(appName)} is ready!`);
+  console.log(`JSS application(s) ${chalk.green(JSON.stringify(appNames))} is ready!`);
```

Comment:
The output of such statement is: "["next-app-test"]" - as an example, that is not right. Also update the unit test to verify exact string